packages/trader/src/AppV2/Containers/Trade/trade-types.tsx

```diff
@@ -125,6 +126,24 @@ const TradeTypes = ({ contract_type, onTradeTypeSelect, trade_types }: TTradeTyp
         }));
     };
 
+    const scrollToSelectedTradeType = () => {
+        setTimeout(() => {
+            let position_x = 0;
+            if (trade_types_ref?.current) {
+                const selected_chip = trade_types_ref?.current.querySelector(
+                    'button[data-state="selected"]'
+                ) as HTMLButtonElement;
+                if (selected_chip) {
+                    position_x = selected_chip?.getBoundingClientRect()?.x || 0;
+                }
+                trade_types_ref.current.scrollBy({
```

Comment:
transition: left 0.3s ease;

can we add the above transition css to make the movement smooth?